includes/Channel.hpp

```diff
@@ -20,6 +20,8 @@
 #include "../includes/Client.hpp"
```

Comment:
REMOVED: #include "../includes/Client.hpp"
This include is not necessary because there is Client class declaration in this file.